samcli/commands/init/init_generator.py

```diff
@@ -37,9 +37,6 @@ def do_generate(location, runtime, dependency_manager, output_dir, name, no_inpu
         "python3.7",
         "python3.6",
         "python2.7",
-        "nodejs",
```

Comment:
Don't remove `nodejs`. It is just a short cut for the latest version.

Reply:
That may've been a mistake 🤦‍♂  Good catch